CHANGELOG.md

```diff
@@ -1,3 +1,115 @@
+# 1.0.0 (2023-09-25)
+
+This release represents a milestone in the development of the extension.
+The package is reorganize to be a collection of more self-contained
+mini-packages. Each with its own set of tests.
+
+Developer documentation and guidelines have been added to aid further
+development. One particular goal is to establish datalad-next as a proxy
+for importing datalad-core functionality for other extensions. Direct imports
+from datalad-core can be minimized in favor of imports from datalad-next
+to enable the identification of functionality needed outside the core package,
```

Comment:
this sentence is somewhat difficult to parse. Maybe you should rephrase it into something actionable to emphasize the envisioned usage a bit more? ("Extensions should minimize imports from datalad-core, and instead import functionality from datalad-next. This can help to identify functionality from datalad-core that is also needed outside of the core package, ..."
But it could also already help to break this sentence into smaller sentences or resolve the nominalization "enable the identification of functionality", which I believe is the phrase posing the biggest hurdle

Reply:
Posted a rephrasing below -- will close this conversation here.